packages/core/saved-objects/core-saved-objects-migration-server-internal/src/core/migrate_raw_docs.ts

```diff
@@ -17,9 +17,11 @@ import type {
   SavedObjectUnsanitizedDoc,
   ISavedObjectsSerializer,
 } from '@kbn/core-saved-objects-server';
-import type { MigrateAndConvertFn } from '../document_migrator/document_migrator';
+import type { VersionedTransformer } from '../document_migrator';
 import { TransformSavedObjectDocumentError } from '.';
 
+type MigrateAndConvertFn = VersionedTransformer['migrateAndConvert'];
```

Comment:
This was the only place where this function type was used, so I inlined it in `VersionedTransformer` instead